app/models/notification.rb

```diff
@@ -25,6 +25,15 @@ def self.mark_all_as_read!
   end
 
   # -- Instance Methods -----------------------------------------------------
+  def project
+    @project ||= Project.new
+  end
+
+  def project=(new_project)
```

Comment:
Do we need to call `save` ?
In the `Node` model we have:
`def project=(new_project); end`